sycl/include/sycl/detail/backend_traits_opencl.hpp

```diff
@@ -143,10 +143,9 @@ namespace pi {
 // Cast for std::vector<cl_event>, according to the spec, make_event
 // should create one(?) event from a vector of cl_event
 template <class To> inline To cast(std::vector<cl_event> value) {
-  sycl::detail::pi::assertion(
-      value.size() == 1,
-      "Temporary workaround requires that the "
-      "size of the input vector for make_event be equal to one.");
+  assert(value.size() == 1 &&
```

Comment:
technically, it's different behavior in release build (where new `assert` will be simply ignored), but I'm fine with that.